src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/DESCryptoServiceProvider.cs

```diff
@@ -16,6 +16,7 @@ public sealed class DESCryptoServiceProvider : DES
 
         public DESCryptoServiceProvider() { }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA5351", Justification = "This is the implementation of DES")]
```

Comment:
Though I think for these errors the suppressions have to be targeting the methods. So we would need assembly level ones for each method. One at the class level wouldn't work.